components/onboarding-modal/OnboardingModal.js

```diff
@@ -0,0 +1,244 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import { Box, Flex } from '@rebass/grid';
+import { graphql } from 'react-apollo';
+import gql from 'graphql-tag';
+
+import Modal, { ModalBody, ModalHeader, ModalFooter } from '../../components/StyledModal';
+import OnboardingNavButtons from './OnboardingNavButtons';
+import OnboardingStepsProgress from './OnboardingStepsProgress';
+import OnboardingContentBox from './OnboardingContentBox';
+import MessageBox from '../../components/MessageBox';
+
+import { getErrorFromGraphqlException } from '../../lib/utils';
+import { getLoggedInUserQuery } from '../../lib/graphql/queries';
+import { Router } from '../../server/pages';
+
+const StepsProgressBox = styled(Box)`
+  min-height: 95px;
+  max-width: 600px;
+
+  @media screen and (max-width: 640px) {
+    width: 100%;
+    max-width: 100%;
+  }
+`;
+
+const params = {
+  0: {
+    height: '114px',
+    src: '/static/images/create-collective/onboardingWelcomeIllustration.png',
+  },
+  1: {
+    height: '112px',
+    src: '/static/images/create-collective/onboardingAdminsIllustration.png',
+  },
+  2: {
+    height: '119px',
+    src: '/static/images/create-collective/onboardingContactIllustration.png',
+  },
+};
+
+class OnboardingModal extends React.Component {
+  static propTypes = {
+    query: PropTypes.object,
+    collective: PropTypes.object,
+    LoggedInUser: PropTypes.object,
+    EditCollectiveMembers: PropTypes.func,
+    EditCollectiveContact: PropTypes.func,
+    show: PropTypes.bool,
+    setShow: PropTypes.func,
+  };
+
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      step: 0,
+      members: [],
+      error: null,
+    };
+  }
+
+  componentDidMount() {
+    this.setStep(this.props.query.step);
+  }
```

Comment:
Seems we don't need to store the step in the state if we just directly read it from the query.

Reply:
If I change this it will require a bit of work to update how the components work so I would do this in another PR.

Could you please let me know whether feedback is blocking/for consideration/a preference?